### PR TITLE
feat: Adds Detokenization reference docs and removes encryption docs

### PR DESCRIPTION
**Pull request by dhudec**
## Description

- Adds Detokenization reference docs section
- Removes all mentions of you encrypt
- Update API Reference sidebar to add a new Reference section to match developers.basistheory.com

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

image

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: source/includes/detokenization/_intro.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+            <a href="#expressions">Detokenization Expressions</a>
+            <a href="#transformations">Transformations</a>
+            <a href="#examples">Examples</a>
+        </span>
+    </span>
+    <img src="/images/detokenization-intro.svg"/>
+</aside>
+
+Detokenization refers to the process by which non-sensitive token identifiers are replaced with the original token data represented by those tokens.
+Basis Theory supports detokenization through our serverless [Reactor](#reactors) platform and the [Proxy](#proxy).
```
Comment (jleon15) [inline source/includes/detokenization/_intro.md:17]: Clicking on reactors or proxy doesn't work as it adds the # under the detokenization path

Comment (jleon15) [inline source/includes/detokenization/_intro.md:17]:
```suggestion
Basis Theory supports detokenization through our serverless [Reactor](/#reactors) platform and the [Proxy](/#proxy).
```

File: source/includes/detokenization/_expressions.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+Any pattern that is wrapped in `{{ }}` will be evaluated for detokenization.
+Detokenization expressions containing non-uuid values, such as `{{non uuid value}}` will be ignored and the expression will not be replaced within the request.
+
+Any `uuid` value contained in a detokenization expression is expected to represent a the `id` of a token within your tenant. 
+If no such token is found with that identifier, the request will be rejected with a 400 error.
+
+If a token is found with the given `uuid` identifier, but the calling application is missing permission to use that 
+token, then the request will be rejected with a 403 error. 
+See the following links for more information about the permissions required to use tokens in a 
+[Reactor](/#reactors-invoke-a-reactor) or [Proxy](/#permissions-proxying-outbound-requests).
```
Comment (jleon15) [inline source/includes/detokenization/_expressions.md:15]: For some reason the Proxy link redirects to the API Reference intro

File: source/includes/detokenization/_expressions.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+<div class="center-column"></div>
+```json
+{
+  "ssn": "111-22-3333"
+}
+```
+
+## Detokenizing Complex Data
+
+Atomic Card or Bank tokens or generic tokens containing complex JSON objects can all be detokenized as well.
```
Comment (jleon15) [inline source/includes/detokenization/_expressions.md:60]:
```suggestion
Atomic Card, Atomic Bank or generic tokens containing complex JSON objects can all be detokenized as well.
```

File: source/includes/detokenization/_expressions.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+Then you can embed the entire [Card](/#atomic-cards-atomic-card-object) object within a request with the detokenization expression:
+
+<div class="center-column"></div>
+```json
+{
+  "card": "{{5c20545b-52dc-4a60-b9b5-5b7c84f22369}}"
+}
+```
+
+will be detokenized into the request:
```
Comment (jleon15) [inline source/includes/detokenization/_expressions.md:87]:
```suggestion
which will be detokenized into the request:
```

File: source/includes/detokenization/_transformations.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+
+Oftentimes data may be tokenized in one format, but you wish to use this data in a different format within a request.
+For example, several pieces of data may be stored together in a complex object such as an Atomic Card token,
+but you wish to only use a single piece of that data within a request.
+
+For this purpose, we offer the ability to apply transformation functions to a detokenization expression. 
+A token can be transformed by specifying a transformation expression after the `|` symbol:   
+`{{ <tokenId> | <transformation> }}`
+
+You can effectively think of this as "piping" the token data into the transformation function in the same way you may be familiar with the "pipe" operator from Unix-like systems.
```
Comment (jleon15) [inline source/includes/detokenization/_transformations.md:11]:
```suggestion
You can effectively think of this as "piping" the token data into the transformation function in the same way you might be familiar with the "pipe" operator from Unix-like systems.
```

File: source/includes/detokenization/_transformations.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+Multiple transformations can be chained together by "piping" the result of each function into the next, applying in order from left to right:  
+`{{ <tokenId> | <transformation1> | <transformation2> | ... }}`
+
+The following sections document the supported library of transformation functions.
+
+## JSON Path
+
+Complex data is typically tokenized as a JSON object within a token's data. In order to facilitate transformations on this
+JSON data, arbitrary [JSON Path](https://tools.ietf.org/id/draft-goessner-dispatch-jsonpath-00.html) expressions can be applied to a token's data:
```
Comment (jleon15) [inline source/includes/detokenization/_transformations.md:20]: I saw this in that site you linked: `This Internet-Draft will expire on 13 January 2021.` Does it mean anything?

File: source/includes/detokenization/_transformations.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+  }
+}
+```
+
+| Expression                                                                                | Result                             |
+|-------------------------------------------------------------------------------------------|------------------------------------|
+| <code>{{d35412f4-9d3b-45d8-b051-fe4b7d4e14c5 &#124; $.bicycle.color}}</code>              | "red"                              |
+| <code>{{d35412f4-9d3b-45d8-b051-fe4b7d4e14c5 &#124; $.bicycle}}</code>                    | { "color": "red", "price": 19.95 } |
+| <code>{{d35412f4-9d3b-45d8-b051-fe4b7d4e14c5 &#124; $.book[0]}}.author</code>             | "Herman Melville"                  |
+| <code>{{d35412f4-9d3b-45d8-b051-fe4b7d4e14c5 &#124; $.book[(@.price < 10)]}}.title</code> | "Moby Dick"                        |
```
Comment (jleon15) [inline source/includes/detokenization/_transformations.md:66]: This is wrong, it is missing the ? and the .title should be inside the expression
```suggestion
| <code>{{d35412f4-9d3b-45d8-b051-fe4b7d4e14c5 &#124; $.book[?(@.price < 10)].title}}</code> | "Moby Dick"                        |
```

File: source/includes/detokenization/_transformations.md [hunk 8]
```diff
@@ -0,0 +1,68 @@
...
+| <code>{{d35412f4-9d3b-45d8-b051-fe4b7d4e14c5 &#124; $.nonexistent</code>                  | `null`                             |
```
Comment (jleon15) [inline source/includes/detokenization/_transformations.md:67]: Missing }}
```suggestion
| <code>{{d35412f4-9d3b-45d8-b051-fe4b7d4e14c5 &#124; $.nonexistent}}</code>                  | `null`                             |
```

File: source/includes/detokenization/_transformations.md [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+| <code>{{d35412f4-9d3b-45d8-b051-fe4b7d4e14c5 &#124; $.book..author</code>                 | <400 Error>                        |
```
Comment (jleon15) [inline source/includes/detokenization/_transformations.md:68]: Missing }}
```suggestion
| <code>{{d35412f4-9d3b-45d8-b051-fe4b7d4e14c5 &#124; $.book..author}}</code>                 | <400 Error>                        |
```

File: source/includes/detokenization/_examples.md [hunk 10]
```diff
@@ -0,0 +1,544 @@
...
+or even use your own custom data schema within a schemaless [generic token](https://docs.basistheory.com/#token-types-token).
+However you choose to tokenize your sensitive data, Basis Theory grants you the flexibility to use this data within
+our serverless Reactor platform or the Proxy.
+
+In this section, we will walk through several examples of how you can detokenize data within Reactors or Proxy requests 
+regardless of how the data was tokenized.
+
+## Prerequisites
+
+If Reactors or Proxy are completely new to you, we recommend you first read our [Reactors](/concepts/what-are-reactors)
```
Comment (jleon15) [inline source/includes/detokenization/_examples.md:14]: Both of these links return 404, these concepts are under https://developers.basistheory.com/
```suggestion
If Reactors or Proxy are completely new to you, we recommend you first read our [Reactors](https://developers.basistheory.com//concepts/what-are-reactors) 
```

File: source/includes/detokenization/_examples.md [hunk 11]
```diff
@@ -0,0 +1,544 @@
...
+      "verification_value": "987",
+      "full_name": "John Doe"
+    },
+    "retained": true
+  }
+}
+```
+
+In this example, we show how you can use an [Atomic Card](https://docs.basistheory.com/#atomic-cards-atomic-card-object) token to create a Spreedly payment method,
+but provide an updated CVC (`987`) that is different from the `cvc` value stored with the token.
```
Comment (jleon15) [inline source/includes/detokenization/_examples.md:226]:
```suggestion
but provide an updated CVC (`987`) that is different from the `cvc` value stored within the token.
```

**Review by jleon15 — CHANGES_REQUESTED** [review]
Looks good, added some comments.

File: source/includes/detokenization/_transformations.md (hunk 5 above)
Comment (dhudec) [reply source/includes/detokenization/_transformations.md:11]: I think `may` sounds better here.

File: source/includes/detokenization/_transformations.md (hunk 6 above)
Comment (dhudec) [reply source/includes/detokenization/_transformations.md:20]: I added a second link to another website. Idk what the date is, but that's the official proposed spec.

File: source/includes/detokenization/_transformations.md (hunk 8 above)
[the remark this answers is not in the document]
Comment (dhudec) [reply source/includes/detokenization/_transformations.md:67]: I'm not sure what you mean... something like `"some text {{ ... }}"`? 

If so, that's an edge case I'm not too worried about explaining here, and it doesn't really fit with the other examples that don't show the context around the expression. Plus I'm already maxed on width in this table without pushing the results out of view.

**Review by jleon15 — DISMISSED** [review]

File: source/includes/api-reference/_proxy.md [hunk 12]
```diff
@@ -59,7 +59,10 @@ Proxy a request to a third party API.
 
 **Authentication**
 
-Proxy requests must be authenticated using an `BT-API-KEY` header (see [Authentication](#authentication)). 
+Proxy requests must be authenticated using an `BT-API-KEY` header (see [Authentication](#authentication)).
```
Comment (koalaty-code) [inline source/includes/api-reference/_proxy.md:62]:
```suggestion
Proxy requests must be authenticated using a `BT-API-KEY` header (see [Authentication](#authentication)).
```

File: source/includes/detokenization/_examples.md [hunk 13]
```diff
@@ -0,0 +1,551 @@
+# Examples
+
+Based on your use specific cases, you may prefer to store your sensitive data with Basis Theory using our
```
Comment (koalaty-code) [inline source/includes/detokenization/_examples.md:3]:
```suggestion
Based on your use case, you may prefer to store your sensitive data with Basis Theory using our
```

**Review by koalaty-code — APPROVED** [review]

**Pull request merged**